modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/BrokerAuthConstants.java

```diff
@@ -52,8 +52,12 @@ private BrokerAuthConstants() {
     public static final String SYSTEM_PARAM_USERS_CONFIG = "broker.users.config";
     // users configuration namespace in users config file.
     public static final String USERS_CONFIG_NAMESPACE = "wso2.broker.users";
-    // user manager property name
-    public static final String PROPERTY_USER_STORE_CONNECTOR = "broker.user.store.connector";
     // user manager authorization id
     public static final String AUTHENTICATION_ID = "AuthenticationId";
+
+    /*
+    JaasAuthenitcator related constants
+     */
+    // user registry property name
+    public static final String PROPERTY_USER_REGISTRY = "broker.user.registry";
```

Comment:
We should not be using single line comments with constants.

Reply:
I was being consistent with the style of this existing class. What is the alternative?

Reply:
fixed so that it will be taken as java doc